plugin.py

```diff
@@ -46,5 +47,56 @@ def on_pre_server_command(self, params: Mapping[str, Any], done_callback: Callab
             done_callback()
         return True
 
+    def on_server_response_async(self, method: str, response: Response) -> None:
+        if method == 'codeAction/resolve':
+            self.maybe_handle_move_code_action(response)
 
+    def maybe_handle_move_code_action(self, response: Response) -> None:
+        session = self.weaksession()
+        if not session:
+            return
+        result = cast(MaybeMoveFunctionCodeAction, response.result)
+        command = result.get('command')
+        if not command:
+            return
+        command_name = command.get('arguments')[0]
+        params = command.get('arguments')[1]
+        function_name = command.get('arguments')[2]
+        if command_name == 'moveFunction':
```

Comment:
Do this check before getting `arguments` since other code actions might not have same arguments structure.

Reply:
No, you need to use `command_name` here...
In any case, ensure that you check length and type of arguments first but there is another issue for that.